Doc/library/__main__.rst

```diff
@@ -116,8 +116,8 @@ Idiomatic Usage
 ^^^^^^^^^^^^^^^
 
 Some modules contain code that is intended for script use only, like parsing
-command-line arguments or fetching data from standard input.  When a module
-like this were to be imported from a different module, for example to unit test
+command-line arguments or fetching data from standard input.  If a module
+like this is imported from a different module, for example to unit test
```

Comment:
I think this would read better as "was imported", to match "would unintentionally execute" later in the same sentence.

Reply:
fair enough, see ef2d00706ad22d998b50d4119f1c4afe0663b93f